packages/datepickers/src/styled/StyledCalendarItem.ts

```diff
@@ -0,0 +1,33 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import styled from 'styled-components';
+import { retrieveComponentStyles, DEFAULT_THEME } from '@zendeskgarden/react-theming';
+
+import { retrieveSpacing } from './utils';
+
+const COMPONENT_ID = 'datepickers.calendar_item';
+
+const StyledCalendarItem = styled.div.attrs({
+  'data-garden-id': COMPONENT_ID
+})<{
+  isSmall?: boolean;
+}>`
+  display: inline-block;
+  position: relative;
+  width: ${retrieveSpacing};
+  height: ${retrieveSpacing};
+  vertical-align: top;
```

Comment:
Just to be clear, when properties are easy to take in all in one glance – as they are here – it does not make sense to worry about breaking out size/color-related styles. So nice setup here. Although, it would even be better if the `retrieveSpacing` were simply included inline where it can be reasoned about in the context of the styled element.